object-construction-checker/tests/mustcall/MustCallChoicePassthroughWrong3.java

```diff
@@ -0,0 +1,29 @@
+// This is a test for what happens when there's a missing MCC return type.
+
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.objectconstruction.qual.*;
+import java.io.*;
+
+class MustCallChoicePassthroughWrong3 {
+    // Both of these verify - it's okay to leave off the MCC param, because the return type is
+    // owning - but the first one leads to imprecision at call sites.
+    static InputStream missingMCC(@MustCallChoice InputStream is) {
```

Comment:
should we get an error here, since there is only one `@MustCallChoice` annotation that is not matched with the return? not a soundness issue, can be handled later

Reply:
I considered adding code to do exactly that, but I decided that it wasn't necessary for soundness so I didn't do it. Hence the comment in this file :)